pom.xml

```diff
@@ -135,6 +135,11 @@
             <plugin>
                 <groupId>com.mycila</groupId>
                 <artifactId>license-maven-plugin</artifactId>
+                <configuration combine.self="override">
+                    <excludes>
+                        <exclude>data/</exclude>
```

Comment:
@janvanmansum we should probably add this to the microservice projects and the archetype as well. @PaulBoon and I experienced problems with the license plugin when we had a `home/` and `data/` folder coming from `debug-reset-apphome.sh` whenever we ran `mvnci`